src/js/_enqueues/wp/updates.js

```diff
@@ -2447,7 +2456,7 @@
 	};
 
 	$( function() {
-		var $pluginFilter        = $( '#plugin-filter, #plugin-information-footer' ),
+		var $pluginFilter        = $( '.plugin-title .row-actions, #plugin-filter, #plugin-information-footer' ),
```

Comment:
```suggestion
		var $pluginFilter        = $( '#plugin-filter, #plugin-information-footer' ),
```
This was some related test code that shouldn't be in the final patch. My mistake in including it in the `.diff`.